cmd/wallet-sdk-gomobile/didcreator/keywrapper.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright Avast Software. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package didcreator
+
+import (
+	arieskms "github.com/hyperledger/aries-framework-go/pkg/kms"
+
+	"github.com/trustbloc/wallet-sdk/cmd/wallet-sdk-gomobile/api"
+)
+
+// gomobileKeyWriterWrapper wraps a gomobile-compatible version of a KeyWriter and translates methods calls to
+// their corresponding Go API versions.
+type gomobileKeyWriterWrapper struct {
+	keyWriter api.KeyWriter
+}
+
+func (g *gomobileKeyWriterWrapper) Create(keyType arieskms.KeyType) (string, []byte, error) {
```

Comment:
doc comment missing? make sure the linter runs this file too

Reply:
The linter does run on that file - it wasn't flagged is because `gomobileKeyWriterWrapper` isn't exported.

Reply:
makes sense